mooncake-wheel/mooncake/mooncake_store_service.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+# mooncake_store_service.py - Integrated Mooncake store service with REST API
+
+import argparse
+import asyncio
+import json
+import logging
+from aiohttp import web
+from mooncake.store import MooncakeDistributedStore
+from mooncake.mooncake_config import MooncakeConfig
+
+class MooncakeStoreService:
```

Comment:
Could we add a few simple configuration examples along with an explanation of what each one does? THX. That might help prevent users from misconfiguring things by accident.

Also, I’m guessing setting the global segment size to 0 might be the right approach here?